Austin_Marga_HW8_1_NEW.ipynb

```diff
@@ -0,0 +1,1230 @@
+{
```

Comment:
<p>I guess we use different notation here for including / excluding borders. You use (0,2pi] I used ]0,2pi].&nbsp;</p><p>So your starting point and your setup width should match. Also you use&nbsp;<code>a=0.01</code>&nbsp;in the next step.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_Calculus/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>